src/STEMMUS_SCOPE.m

```diff
@@ -568,7 +550,13 @@
 hCHK = zeros(1, NN);
 TIMELAST = 0;
 
-% Cause the start of simulation period is from 0mins, while the input data start from 30mins.
+% Convert unit to Centimeter-Gram-Second system
+% see issue 188 to refactor these lines
+HR_a = 0.01 .* (ForcingData.RH_msr);
+U = 100 .* (ForcingData.WS_msr);
+TopPg = 100 .* (ForcingData.Pg_msr);
+
+% the start of simulation period is from 0mins, while the input data start from 30mins.
 tS = DURTN / Delt_t;
```

Comment:
```matlab
tS = DURTN / Delt_t;
```
It seems that `tS` is a synonym for `Dur_tot`...

see:
```matlab
DURTN = TimeProperties.DELT * TimeProperties.Dur_tot; % Duration of simulation period;
Delt_t = TimeProperties.DELT; % Duration of time step [Unit of second]
```

And then the following line can also be refactored:
```
TEND = TIME + DURTN; % Time to be reached at the end of simulation period
```
to

```matlab
TEND = TIME + TimeProperties.DELT * TimeProperties.Dur_tot;
```

(TEND is not a global, and tS isn't either).

So I think this is a good chance to get rid of two more excess variables (DURTN, tS) :sweat_smile:

Reply:
Done!